### ISSUE TITLE
Tagging error

### ISSUE THREAD

**Issue by zzj0402** [spencermountain/compromise#585]
```
This is the second test message.
```
"test message"
returns

```
bestTag: "Infinitive"
implicit: ""
normal: "test"
spaceAfter: ""
spaceBefore: " "
tags: (4) ["Infinitive", "PresentTense", "Verb", "VerbPhrase"]
text: "test"
```
```
bestTag: "Infinitive"
implicit: ""
normal: "message"
spaceAfter: ""
spaceBefore: " "
tags: (4) ["Infinitive", "PresentTense", "Verb", "VerbPhrase"]
text: "message."
```

**Comment by spencermountain** [comment]
thanks

**Comment by zzj0402** [comment]
Input: 
["In statistical classification, including machine learning, two main approaches are called the generative approach and the discriminative approach."  ](https://en.wikipedia.org/wiki/Generative_model)  

Tags mislabeled:
```
bestTag: "Actor"
implicit: ""
normal: "generative"
spaceAfter: ""
spaceBefore: " "
tags: (3) ["Actor", "Noun", "Singular"]
text: "generative"

bestTag: "Actor"
implicit: ""
normal: "discriminative"
spaceAfter: ""
spaceBefore: " "
tags: (3) ["Actor", "Noun", "Singular"]
text: "discriminative"
```

**Comment by spencermountain** [comment]
fixed in `12.3.0`.
thanks
